packages/appium/lib/extension/manifest-migrations.js

```diff
@@ -0,0 +1,94 @@
+import log from '../logger';
+
+/**
+ * This contains logic to migrate old versions of `extensions.yaml` to new versions.
+ *
+ * @module
+ */
+
+/**
+ * Collection of functions to migrate from one version to another
+ * @type {{[P in keyof ManifestDataVersions]?: Migration}}
+ */
+const Migrations = {
+  /**
+   * If `installPath` is missing from the `ExtManifest` for any extension, return `true`.
+   *
+   * We cannot easily determine the path of any given extension here, but returning `true`
+   * will cause the `Manifest` to sync with the filesystem, which will populate the missing field.
+   *
+   * @type {Migration}
+   */
+  3: (data) => {
+    let shouldSync = false;
```

Comment:
do we need this intermediate variable at all?

Reply:
not technically, no, but it's just for clarity of intent